deb/build/build.sh

```diff
@@ -22,10 +22,17 @@ sed -i.bak -e 's/^\s*$/./' -e 's/^/ /' $DESCRIPTION_FILE
 # Rewrite the file
 mv "$DESCRIPTION_FILE.bak" "$DESCRIPTION_FILE"
 
+# Define changelog link for weekly releases
+CHANGELOG_LINK=https://jenkins.io/changelog/#v${VERSION}
```

Comment:
```suggestion
CHANGELOG_LINK=https://jenkins.io/changelog${RELEASELINE}/#v${VERSION}
```
Wouldn't it be better to use RELEASELINE instead? My main concern is, as far as I know jenkins.io/changelog-* generation is not related to release process, so let image that we have an experimental release, the description will be broken

Reply:
As far as I can tell, `RELEASELINE` has 5 allowed values (in order from most frequently used to least frequently used):

* (empty) - weekly
* stable
* stable-rc
* experimental
* rc

The changelogs are maintained on jenkins.io for two of those lines:

* (empty) - weekly
* stable

I'm fine with using RELEASELINE since it will remove logic and pattern matching from the script.

We have ongoing efforts to automate the changelog generation process.  Until the process is completely automated, the changelog link would be unsatisfied until the changelog is published.  

In the last several weeks, changelogs have been published within a few hours of the release or at the time of release.  In the last 6-12 months, the longest delay between Jenkins release and changelog publication has consistently happened within 1-2 days of release.

I think that a potentially broken link to a changelog is still significantly better than the current non-existing changelog information in the Debian/Ubuntu packaging.

Reply:
See 8bb1331b4b15e2209c7cfc531e49b49c2374afc8 for the revision